src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -111,7 +111,7 @@ private void export(Path file, FileChooser.ExtensionFilter selectedExtensionFilt
             // All entries
             entries = stateManager.getActiveDatabase()
                                   .map(BibDatabaseContext::getEntries)
-                                  .orElse(Collections.emptyList());
```

Comment:
Is this change needed? I checked the caller - and it seems it is passed to `org.jabref.logic.exporter.Exporter#export(org.jabref.model.database.BibDatabaseContext, java.nio.file.Path, java.util.List<org.jabref.model.entry.BibEntry>, java.util.List<java.nio.file.Path>, org.jabref.logic.journals.JournalAbbreviationRepository)`, which requires a list and nothing observable.

I would even try to use `toList()` since the list should not be modified (should it?)

Reply:
I believe I just did this because the function signature changed (from `Optional<List<..>>` to `Optional<ObservableList<...>>` and I was trying to minimally fix to comply. Will add in the `toList` call and change it back.

Reply:
Per my earlier comment - 

> This push changes the method signature of `getEntries()` (since it was already an `ObservableList` and something needed to be able to bind to it). Removing the `.map(List::stream).map(Stream::toList)` causes compile errors and seems to be the reason tests are currently failing. I've changed it to instead do `.orElse(FXCollections.emptyObservableList())` and that seems to address the compile issue.